main.tf

```diff
@@ -325,7 +325,7 @@ resource "aws_cloudwatch_log_group" "session_logging" {
 resource "aws_ssm_document" "session_logging" {
   count = var.session_logging_enabled && var.create_run_shell_document ? 1 : 0
 
-  name          = "SSM-SessionManagerRunShell"
+  name          = "SSM-SessionManagerRunShell-${var.stage}"
```

Comment:
I think instead of creating these for each stage that you're using, I would suggest that you create this ssm_document resource externally to this module and pass `create_run_shell_document = false`. Same goes for the above KMS key, which I will comment on.

Reply:
@nordinh similar to the above -- Mind reverting and utilizing `create_run_shell_document = false` as mentioned above?